fe1-web/src/features/notification/navigation/NotificationNavigation.tsx

```diff
@@ -0,0 +1,33 @@
+import { createStackNavigator } from '@react-navigation/stack';
+import React from 'react';
+
+import STRINGS from 'resources/strings';
+
+import NotificationScreen from '../screens/NotificationScreen';
+import SingleNotificationScreen from '../screens/SingleNotificationScreen';
+import { NotificationStackParamList } from './NotificationStackParamList';
+
+const NotificationStackNavigator = createStackNavigator<NotificationStackParamList>();
+
+const NotificationNavigation = () => {
+  return (
+    <NotificationStackNavigator.Navigator
+      initialRouteName={STRINGS.notification_navigation_tab_notifications}>
+      <NotificationStackNavigator.Screen
+        name={STRINGS.notification_navigation_tab_notifications}
+        component={NotificationScreen}
+        options={{
+          title: STRINGS.notification_navigation_tab_notifications_title,
+          // show not back button for this screen
+          headerLeft: () => null,
```

Comment:
I'm not sure I understand this.

Reply:
I want to prevent this arrow from showing.
<img width="272" alt="Screenshot 2022-05-03 at 08 35 49" src="https://user-images.githubusercontent.com/2634739/166413136-8522212e-49d2-4603-8c75-078683e16c1b.png">

I now updated the comment to
```
// show no back button for this screen in the top navigation bar
// the back button should only be used from notification detail/single views
// to get back to this screen but not to go from this screen "back" to anywhere
```

Is this clearer?